### DISCUSSION TITLE
Have you based this profile in an existing one?

### DISCUSSION THREAD

**Discussion by ricardomfmsousa** [suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config#6] [General]
Hello, thank you for your time and effort to build this profile!
I've been investigating 3D printing for a couple of years, and finally I got my first printer one month ago - the Ender 3 v3 SE.

I've realized that no good slicer has it's profile unfortunately, and Creality's one is lacking many features, pretty buggy and slow (at least on Linux).

I've read somewhere that a good base for the SE profile would be the Ender 3 S1, since they are quite similar in some key hardware. I'm just wondering if that's what you've used.

**Comment by suchmememanyskill** [comment]
I first tried porting all relevant settings from creality slicer to a from scratch prusaslicer profile, then filling the gaps for settings without a direct translation from the s1 profile.

If you're curious, prusaslicer in expert mode has a button on the top to compare profiles across different printers
